core/data_models.py

```diff
@@ -38,6 +38,10 @@ class SociosBrasilEmpresaMixin:
     def is_headquarter(self):
         return self.cnpj[:12].endswith("0001")
 
+    @property
+    def name(self):
+        return self.razao_social if self.razao_social else self.cnpj
```

Comment:
`nome_fantasia` deve ter precedência sobre o CNPJ e também devemos tratar casos em que o valor é uma string vazia ou espaço.

```suggestion
        keys_to_try = ("razao_social", "nome_fantasia", "cnpj")
        for key in keys_to_try:
            value = str(getattr(self, key, None) or "").strip()
            if value:
                return value
```